src/components/inputForm/InputForm.tsx

```diff
@@ -1,17 +1,51 @@
-import { PLACE_HOLDER } from '../../constants';
+import { useRef } from 'react';
 import Icon from '../icon/Icon';
+import smileSVG from '../../assets/smile.svg';
+import { getRandomUser } from '../../utils/getRandomUser';
+import { PLACE_HOLDER, POST } from '../../constants';
+import { st, classes as generalClasses } from '../../style/general.st.css';
+import { classes } from './InputForm.st.css';
+import type { Comment } from '../../types';
 
 interface IInputForm {
-    onSubmit: (value: string) => void;
+    likeComment: (date: number, userName: string) => void;
+    onSubmit: (comment: Comment) => void;
 }
 
-const InputForm = ({ onSubmit }: IInputForm) => (
-    <form>
-        <div>
-            <Icon className={''} img={'smileSVG'} />
-            <input type="text" placeholder={PLACE_HOLDER} />
-        </div>
-        <button type="submit">Post</button>
-    </form>
-);
+const InputForm = ({ likeComment, onSubmit }: IInputForm) => {
+    const inputElement = useRef<HTMLInputElement>(null);
+    const formElement = useRef<HTMLFormElement>(null);
+    const submitForm = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+
+        if (inputElement.current instanceof HTMLInputElement && inputElement.current.value.trim()) {
+            const randomUser = getRandomUser();
+            const newComment: Comment = {
+                user: randomUser,
+                content: inputElement.current.value,
+                creationDate: Date.now(),
+                LikedBy: [],
+                likeComment,
+            };
+            onSubmit(newComment);
+        }
+        formElement.current?.reset();
```

Comment:
why not add ''&& current'' to the condition

Reply:
I want to reset the form even if the value is invalide

Reply:
so its out side the if